src/app/lib/config/toggles/const.js

```diff
@@ -9,7 +9,6 @@ export default {
   LIVE_RADIO_SCHEDULE: 'liveRadioSchedule',
   MEDIA_PLAYER: 'mediaPlayer',
   MOST_READ: 'mostRead',
-  NAVIGATION_ON_ARTICLES: 'navOnArticles',
```

Comment:
Given that we're deleting this toggle, should we also remove it from iSite (test & live) to avoid confusion?

Reply:
Yea should clean up afterwards. May need this to actually go to Live before we can do that though as removing the toggle may cause the nav bar to disappear without removing these checks for it.